### DISCUSSION TITLE
Trying to work with the dreadful User Experience

### DISCUSSION THREAD

**Discussion by marcuslvr-srs** [community/community#38811] [New to GitHub]
I am sorry but the User experience for this product is awful. With the popularity of this solution i assume i am alone. I have been forced by my company to use this for the last year and to my surprise the majority of my collogues agree.
My problem today is that i went back into github.com for the 2nd time today and got the following screen without any explaination as to why this was suddenly put in my way:
![image](https://user-images.githubusercontent.com/87336411/201202910-8e5a11e2-f1b1-49ae-84a9-ba9c2c56830e.png)

My other big issue that i laugh about - as example of how bad it is - that when i create a new repo any tool i use with the repo will fail silently unless i add a - un-needed file i.e., i now know to add a readme file.

Also, i find that the UI is super counter-intuitive (biggest polite insult of professional UI) and i talk with others about how bad it is with others and i find i am not the odd one out but instead the majority view. 

By the way 2 positives a) The GIT command line works well and makes sense b) Working from Visual Studio works well

Love to hear of any solutions and any plans to revisit the user experience to help very technical user discover and enjoy your product, i realize at this point that is a colossal task but has to start as sometime. 

Something that may not be obvious, even with my vehement opinions, i would be interested / keen in working with you to address these challenges.

**Comment by citrushouse** [comment]
I joined GitHub this month and I have to disagree with that, I think it's a nice UX to confirm security settings --it's not like you have to insert anything. From colors to layout, most of GitHub's UI and UX is subjective because there's nothing blatantly "wrong." "keen in working with you to address these challenges" gave me a chuckle though, lol 😂

----

I always wanted to say this about GitHub though: Beyond colors and layout, I feel like GitLab and Azure DevOps have much better CI/CD features and a more intuitive approach compared to GitHub.

![Screenshot 2022-11-10 152308](https://user-images.githubusercontent.com/116568519/201209730-4f65245c-8125-4115-8959-bbca84ee8262.png)

![Screenshot 2022-11-10 152439](https://user-images.githubusercontent.com/116568519/201210316-38e4785e-5667-4274-b76f-b798050922e7.png)
